_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -4,6 +4,8 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+* Cockpit single sign-on via "Web Console" button. For more information how to set it up, head over to [Remote Execution](/plugins/foreman_remote_execution/1.7/index.html#3.6CockpitIntegration) manual.
```

Comment:
Oops, i missed that this was the nightly manual, could you place this in the 1.24 manual instead? Sorry for that.

Reply:
Yeah I was wondering why it was so empty :-) Done.